garrysmod/lua/includes/modules/hook.lua

```diff
@@ -1,158 +1,101 @@
-local gmod = gmod
-local debug = debug
 local pairs = pairs
-local getmetatable = getmetatable
-local setmetatable = setmetatable
+local isstring = isstring
+local isfunction = isfunction
+
+local function getGAMEMODE()

```

Comment:
overkill change for this one, wasn't required tbh

Reply:
https://github.com/Facepunch/garrysmod/blob/a254b97be82a02899bf3093f93fac277cc8a128f/garrysmod/gamemodes/base/gamemode/cl_init.lua#L73-L79
but I could revert this change